src/bundle/Resources/config/ui/mappers.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
+    ezrichtext.custom_tags.translation_domain: 'custom_tags'
+    ezrichtext.custom_styles.translation_domain: 'custom_styles'
+    ezrichtext.online_editor.translation_domain: 'online_editor'
+
+services:
+    _defaults:
+        autowire: true
+        public: false
+
+    # RichText Custom Tags UI config attribute type mappers
+    EzSystems\EzPlatformRichText\Configuration\UI\Mapper\CustomTag\CommonAttributeMapper:
+        tags:
+            - { name: ezrichtext.configuration.custom_tag.mapper, priority: 0 }
+
+    EzSystems\EzPlatformRichText\Configuration\UI\Mapper\CustomTag\ChoiceAttributeMapper:
+        parent: EzSystems\EzPlatformRichText\Configuration\UI\Mapper\CustomTag\CommonAttributeMapper
+        autowire: true
```

Comment:
```suggestion
```

`true` is the default value for `autowire` option in this file.

Reply:
> `true` is the default value for `autowire` option in this file.

```
Attribute "autowire" on service "EzSystems\EzPlatformRichText\Configuration\UI\Mapper\CustomTag\ChoiceAttributeMapper" cannot be inherited from "_defaults" when a "parent" is set. Move   
  your child definitions to a separate file or define this attribute explicitly
```
(the former one seems to be an overkill here)